### PR TITLE
[FIX] pstats_print2list: Fix the python3 import

### PR DESCRIPTION
**Pull request by JesusZapata**
Adding support to python3 over the `pstats_print2list.py`

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
This pull request enable the python3 over the `pstats_print2list` package

File: pstats_print2list/pstats_print2list.py [hunk 1]
```diff
@@ -5,7 +5,16 @@
...
+try:
+    import StringIO
+except ImportError:
+    from io import StringIO
+
+
+try:
+    basestring
+except NameError:
+    basestring = str
```
Comment (moylop260) [inline pstats_print2list/pstats_print2list.py]: Use typestring like as your use it from pylint-odoo

File: pstats_print2list/pstats_print2list.py [hunk 2]
```diff
@@ -5,7 +5,16 @@
 import os
 import pstats
 import re
-import StringIO
+try:
+    import StringIO
```
Comment (moylop260) [inline pstats_print2list/pstats_print2list.py]: Is not from StringIo import StringIO to have the same effect?

Comment (JesusZapata) [reply pstats_print2list/pstats_print2list.py]: Done

File: pstats_print2list/pstats_print2list.py (hunk 1 above)
Comment (JesusZapata) [reply pstats_print2list/pstats_print2list.py]: Done

**Pull request merged**